oracle-linux-image-tools/cloud/vagrant-virtualbox/files/vagrant-common.sh

```diff
@@ -0,0 +1,139 @@
+  chmod 600 /home/vagrant/.ssh/authorized_keys
+  chown -R vagrant:vagrant /home/vagrant/.ssh
+
+  # Fix for issue #76, regular users can gain admin privileges via su
+  ex -s /etc/pam.d/su <<'EOF'
+/^account\s\+sufficient\s\+pam_succeed_if.so uid = 0 use_uid quiet$/
+:append
+# allow vagrant to use su, but prevent others from becoming root or vagrant
+account         [success=1 default=ignore] \\
+                                pam_succeed_if.so user = vagrant use_uid quiet
+account         required        pam_succeed_if.so user notin root:vagrant
+.
+:update
+:quit
+EOF
+
+  echo 'vag' > /etc/yum/vars/infra
+
+  # Configure grub to wait just 1 second before booting
+  sed -i 's/^GRUB_TIMEOUT=[0-9]\+$/GRUB_TIMEOUT=1/' /etc/default/grub
+  grub2-mkconfig -o /boot/grub2/grub.cfg
+
+  # Blacklist the floppy module to avoid probing timeouts
+  echo blacklist floppy > /etc/modprobe.d/nofloppy.conf
+  chcon -u system_u -r object_r -t modules_conf_t /etc/modprobe.d/nofloppy.conf
+
+  # Customize the initramfs
+  # Enable VMware PVSCSI support for VMware Fusion guests.
+  echo 'add_drivers+=" mptspi "' > /etc/dracut.conf.d/vmware-fusion-drivers.conf
+  restorecon /etc/dracut.conf.d/vmware-fusion-drivers.conf
+  # There's no floppy controller, but probing for it generates timeouts
+  echo 'omit_drivers+=" floppy "' > /etc/dracut.conf.d/nofloppy.conf
+  restorecon /etc/dracut.conf.d/nofloppy.conf
+  # Regenerate initrd
+  local current_kernel=$(uname -r)
+  ${DRACUT_CMD} -f "/boot/initramfs-${current_kernel}.img" "${current_kernel}"
+
+  # Set SELinux to enforcing
+  sed -i -e 's/^SELINUX\s*=.*/SELINUX=enforcing/' /etc/selinux/config
+
+  # Install additional release packages and enable repos
+  yum install -y ${YUM_VERBOSE} wget \
+    oracle-softwarecollection-release-el7
+  yum-config-manager --enable  ol7_addons >/dev/null
+  yum-config-manager --enable  ol7_optional_latest >/dev/null
+
+  # Install developer release packages and enable repos
+  if [[ "${VAGRANT_DEVELOPER_REPOS,,}" = "yes" ]]; then
+    yum install -y ${YUM_VERBOSE} oracle-epel-release-el7 \
+      oraclelinux-developer-release-el7
+    yum-config-manager --enable  ol7_preview >/dev/null
+    yum-config-manager --enable  ol7_developer >/dev/null
+    yum-config-manager --enable  ol7_developer_EPEL >/dev/null
+  fi
+
+  # Add login banner
+  echo "
+Welcome to Oracle Linux Server release `cat /etc/os-release | grep ^VERSION= | grep -o "[0-9].[0-9]"` (GNU/Linux $(uname -r))
+
+The Oracle Linux End-User License Agreement can be viewed here:
+
+  * /usr/share/eula/eula.en_US
+
+For additional packages, updates, documentation and community help, see:
+
+  * http://yum.oracle.com/
+  " > /etc/motd
+
+}
+
+#######################################
+# Cleanup module
+# Globals:
+#   None
+# Arguments:
+#   None
+# Returns:
+#   None
+#######################################
+vagrant::cleanup()
+{
+  distr::remove_rpms usermode \
+    rhn\* \
+    psmisc \
```

Comment:
I'm genuinely curious to know why we're removing these packages, especially `psmisc` (which contains `killall`) and `m2crypto` for OpenSSL support in Python.

Reply:
For no particular reason, except it is what we are currently doing in the released boxes.